client/packages/dashboard/src/widgets/ReplenishmentWidget.tsx

```diff
@@ -30,6 +41,28 @@ export const ReplenishmentWidget: React.FC<PropsWithChildrenOnly> = () => {
     error: requisitionCountError,
   } = useDashboard.statistics.requisitions();
 
+  const { customDate, urlQueryDateTime } = useFormatDateTime();
+
+  const getTodayUrlQuery = () => {
+    const startOfDay = DateUtils.startOfDay(new Date());
+    const endOfDay = DateUtils.endOfDay(new Date());
+
+    return `${customDate(
+      startOfDay,
+      urlQueryDateTime
+    )}${RANGE_SPLIT_CHAR}${customDate(endOfDay, urlQueryDateTime)}`;
+  };
+
+  const getThisWeekUrlQuery = () => {
+    const previoudMonday = DateUtils.previousMonday(new Date());
+    const endOfWeek = DateUtils.endOfWeek(new Date());
+
+    return `${customDate(
+      previoudMonday,
+      urlQueryDateTime
+    )}${RANGE_SPLIT_CHAR}${customDate(endOfWeek, urlQueryDateTime)}`;
+  };
```

Comment:
Couple of minor things
- the typo in `previoudMonday` variable
- adding the startOfDay and endOfDay
- possibly including Sunday as well ( the `weekStartsOn` option: date-fns defaults to the week starting on Sunday 🙄 )

```suggestion
  const getThisWeekUrlQuery = () => {
    const previousMonday = DateUtils.startOfDay(
      DateUtils.previousMonday(new Date())
    );
    const endOfWeek = DateUtils.endOfDay(
      DateUtils.endOfWeek(new Date(), { weekStartsOn: 1 })
    );

    return `${customDate(
      previousMonday,
      urlQueryDateTime
    )}${RANGE_SPLIT_CHAR}${customDate(endOfWeek, urlQueryDateTime)}`;
  };
```